client/src/main/java/org/apache/rocketmq/client/producer/LocalTransactionExecutor.java

```diff
@@ -18,6 +18,6 @@
 
 import org.apache.rocketmq.common.message.Message;
 
-public interface LocalTransactionExecuter {
-    public LocalTransactionState executeLocalTransactionBranch(final Message msg, final Object arg);
+public interface LocalTransactionExecutor {
```

Comment:
1) This project is still at incubation stage, if we do not correct things now, we may never ever get the chance to fix them afterwards.
2) Transaction related feature is not currently implemented, IMO, we may fix this interface before it's widely used.